### PR TITLE
Add Cut/Copy/Paste context menus on input fields (#336)

### PR DESCRIPTION
**Pull request by jmsaulnier**
**What current issue(s) from Trello/Github does this address?**
#336

**What problem does this PR solve?**
Right Click on input doesn't do anything. You should see a menu with cut / copy / past / select all options

**How did you solve this problem?**
Add Electron menu input config

**How did you make sure your solution works?**
Tests

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

### CODE REVIEW

**Comment by dvdschwrtz** [comment]
I am not seeing any difference to the copy and paste functionality when I use these changes. What does it mean that it adds a context menu? @evgenyboxer can you also take a look at this

**Comment by jmsaulnier** [comment]
@dvdschwrtz I updated my PR comment, a right click on input elements should display a menu with cut / copy / past / select all options

File: main.js [hunk 1]
```diff
@@ -80,6 +80,17 @@ app.on('ready', () => {
       Menu.setApplicationMenu(menu)
     }
 
+    const inputMenu = Menu.buildFromTemplate([
```
Comment (dvdschwrtz) [inline main.js]: per our slack discussion, please change to only allow for 'paste'. thanks and look forward to working with you on the wallet :)

Comment (jmsaulnier) [reply main.js]: @dvdschwrtz done 👌

**Pull request merged**